crates/voicevox_core_java_api/lib/src/main/java/jp/hiroshiba/voicevoxcore/VoicevoxCoreInfo.java

```diff
@@ -0,0 +1,93 @@
+package jp.hiroshiba.voicevoxcore;
+
+import com.google.gson.Gson;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import jakarta.annotation.Nonnull;
+
+/** VOICEVOX CORE自体の情報。 */
+public class VoicevoxCoreInfo extends Dll {
+  /**
+   * ライブラリのバージョン。
+   *
+   * @return ライブラリのバージョン。
+   */
+  @Nonnull
+  public static String getVersion() {
+    String version = rsGetVersion();
+    if (version == null) {
+      throw new NullPointerException("version");
+    }
+    return version;
```

Comment:
ここはJNI実装を信頼してもいいと思います。

```suggestion
    return rsGetVersion();
```